ntbs-service/Startup.cs

```diff
@@ -39,12 +40,23 @@ public void ConfigureServices(IServiceCollection services)
                 options.UseSqlServer(Configuration.GetConnectionString("ntbsContext"))
             );
 
+            var auditDbConnectionString = Configuration.GetConnectionString("auditContext");
+
+            services.AddDbContext<AuditDatabaseContext>(options =>
+                options.UseSqlServer(auditDbConnectionString)
+            );
+
+            if (Configuration.GetValue<bool>(Constants.AUDIT_ENABLED_CONFIG_VALUE)) {
+                services.AddEFAuditer(auditDbConnectionString);
```

Comment:
Now added in here as you suggested, which makes a lot more sense :)
I decided to add a flag to disable auditing - we still need to register the auditDatabaseContext above as it gets DI-ed, but when you set auditenabled = false in settings we don't register the EFAuditer so no automatic logging is performed

Reply:
nice!